Formula/node_exporter.rb

```diff
@@ -23,18 +23,22 @@ def install
     system "go", "build", "-ldflags", ldflags.join(" "), "-trimpath",
            "-o", bin/"node_exporter"
     prefix.install_metafiles
+
+    (etc/"node_exporter.args").write <<~EOS
+      --web.listen-address=127.0.0.1:9101
```

Comment:
This changes the default behavior so I think creating an empty file would cause less hidden problems.